### PR TITLE
fix: update edu app launcher

### PR DESCRIPTION
**Pull request by fregataa**
1. Request eduapp credential with query string
Previously, we request credential with `stoken` that is embedded in json body.
Change `stoken` to be embedded in query string since the GET request does not contain the json data.

2. Handle `bokff` image apps
edu app launcher uses predefined session templates to create session and connect to apps. Since the name of the app should be the name of the template and recently we added new session templates with new images, we need to support the new session templates which has name prefix `bokff`

3. Sleep before connect to `rstudio` app
edu applauncher needs to wait(sleep) before connecting `rstudio` app because `rstudio` bootstraps for a long time. The issue should be resolved with more detail investigation in the future.

# How to test
[Follow this link](https://lablup.sharepoint.com/:fl:/s/b334cd9f-0792-4007-b883-78ef6adfbd9b/EQDtlPsQToxBmeDOi1zljvkBOBxPK00g8cciC60spr_vbA?e=71d8tx&nav=cz0lMkZzaXRlcyUyRmIzMzRjZDlmLTA3OTItNDAwNy1iODgzLTc4ZWY2YWRmYmQ5YiZkPWIlMjFLZDR5eDFONkUwMnN5cG5Kd3dXTnhMbHFzYU12SDlkS2xLeTZHa29tcVUyUTBZcTdyUFJDVEpZdkxqVktWU2h3JmY9MDFLU1JMSFhZQTVXS1BXRUNPUlJBWlRZR09STk9PTERYWiZjPSUyRiZhPUxvb3BBcHAmcD0lNDBmbHVpZHglMkZsb29wLXBhZ2UtY29udGFpbmVyJng9JTdCJTIydyUyMiUzQSUyMlQwUlRVSHhzWVdKc2RYQXVjMmhoY21Wd2IybHVkQzVqYjIxOFlpRkxaRFI1ZURGT05rVXdNbk41Y0c1S2QzZFhUbmhNYkhGellVMTJTRGxrUzJ4TGVUWkhhMjl0Y1ZVeVVUQlpjVGR5VUZKRFZFcFpka3hxVmt0V1UyaDNmREF4UzFOU1RFaFlORFJHV1ZnelFVSkVNak0xUVZrMk4wSklOMDFUVTBkWVdFUSUzRCUyMiUyQyUyMmklMjIlM0ElMjI2ZmExZDIxMy04M2RmLTQyYzItYTYwYS1mNGYzYjVhMzZlZDYlMjIlN0Q%3D)

**Checklist:** (if applicable)

- [x] Minium required manager version
- [x] Specific setting for review (eg., KB link, endpoint or how to setup)

### CODE REVIEW

File: src/components/backend-ai-edu-applauncher.ts [hunk 1]
```diff
@@ -473,20 +484,32 @@ export default class BackendAiEduApplauncher extends BackendAIPage {
...
-        requestedApp = 'jupyterlab';
-      }
       this._openServiceApp(sessionId, requestedApp);
     }
   }
 
   async _openServiceApp(sessionId, appName) {
     this.appLauncher.indicator = await globalThis.lablupIndicator.start();
-    console.log(`launching ${appName} from session ${sessionId} ...`);
+    let requestedApp = appName;
```
Comment (agatha197) [inline src/components/backend-ai-edu-applauncher.ts:493]: Not that important thing, but can you unify the naming convention of appName? `eduAppnamePrefix` and `appname` / `eduAppNamePrefix` and `appName`.

**Review by agatha197 — CHANGES_REQUESTED** [review]

File: src/components/backend-ai-edu-applauncher.ts (hunk 1 above)
Comment (lizable) [reply src/components/backend-ai-edu-applauncher.ts:493]: Changed `eduAppnamePrefix` to `eduAppNamePrefix`, following lowerCamelCase rule.

**Review by lizable — APPROVED** [review]
LGTM.

File: src/components/backend-ai-edu-applauncher.ts (hunk 1 above)
Comment (fregataa) [reply src/components/backend-ai-edu-applauncher.ts:493]: updated webserver in https://github.com/lablup/backend.ai/pull/1738

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**